src/src/MetricsWrapper.js

```diff
@@ -132,6 +132,36 @@ export class MetricsWrapper {
     return counts;
   }
 
+  pprintAwsResourceCountsForCapability(capabilityId) {
+    const metric = this.metrics.get(MetricsWrapper.ResourceCountsKey);
+
+    const counts = metric.capabilitiesMap.get(capabilityId);
+    if (counts === undefined) {
+      return false;
+    }
+    let total = 0;
+    for (let [key, value] of counts) {
+      total += value;
+    }
+    return true;
+  }
```

Comment:
not completely sure what this function does. Is it debug code that shouldve been removed?